src/lib/instructors.ts

```diff
@@ -36,3 +39,26 @@ export async function loadInstructor(slug: string) {
 
   return instructor
 }
+
+const sanityInstructorHash = {
+  'stephanie-eckles': stephanieEcklesQuery,
+}
+
+type SelectedInstructor = keyof typeof sanityInstructorHash
+
+const canLoadSanityInstructor = (
+  selectedInstructor: string,
+): selectedInstructor is SelectedInstructor => {
+  const keyNames = Object.keys(sanityInstructorHash)
+
+  return keyNames.includes(selectedInstructor)
+}
+
+export const loadSanityInstructor = async (selectedInstructor: string) => {
+  if (!canLoadSanityInstructor(selectedInstructor)) return
+
+  const query = sanityInstructorHash[selectedInstructor]
+  if (!query) return
+
+  return await sanityClient.fetch(query)
+}
```

Comment:
@theianjones helped me implement this. 

`loadSanityInstructor` uses a type guard (`canLoadSanityInstructor`) to check that the instructor slug (the one that will be passed in via `getServerSideProps`) is present in the defined `sanityInstructorHash` as a key. If it is, that key is associated with the query that it will call. 

This makes sure that we are only querying for instructor data on the pages that we want to load sanity data onto.